ott/tools/plot.py

```diff
@@ -76,9 +93,11 @@ def couplings(arg: Union[transport.Transport, jnp.ndarray],
 
   if isinstance(arg, transport.Transport):
     ot = arg
-    return _couplings(ax, ot.geom.x, ot.geom.y, ot.a, ot.b, ot.matrix, **kwargs)
+    _couplings(ax, ot.geom.x, ot.geom.y, ot.a, ot.b, ot.matrix, **kwargs)
+    return ax
```

Comment:
why not returning ax on line 80 instead as the return of the _couplings function. Then you do not have to change those lines here

Reply:
done.